### PR TITLE
Add some resources to Readme

### PR DESCRIPTION
**Pull request by ldotto46**

### CODE REVIEW

**Review by PratikGarai — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,14 @@
-# Solidity-Beginner-to-Advanced
+# 👋 Solidity-Beginner-to-Advanced
 A repo for solidity snippets and helpful resources
+- [What is a Smart Contract?](https://ethereum.org/en/developers/docs/smart-contracts/)
+
+## 🔍 Cryptography
```
Comment (PratikGarai) [inline README.md:5]: @ldotto46  Can you kindly change the attached links under this section to open the English version by default?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,14 @@
...
+- [What is a SHA-256](https://academy.bit2me.com/sha256-algoritmo-bitcoin/)
+- [HASH](https://academy.bit2me.com/que-es-hash/)
+- [NONCE?](https://academy.bit2me.com/que-es-nonce/)
+
+# 🏗️ 🔨 Built With
+- [SOLIDITY](https://docs.soliditylang.org/en/v0.8.9/#): Solidity is an object-oriented, high-level language for implementing smart contracts. It is designed to target the Ethereum Virtual Machine (EVM). There are many other blockchains (other than Ethereum) that are EVM compatible. If you’re interested to learn more I recommend reading the Introduction to Smart Contracts section in the Solidity docs.
+
```
Comment (PratikGarai) [inline README.md:19]: Optional : Include section for links to deployment tools (remix, truffle, etc)

**Review by PratikGarai — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ldotto46) [reply README.md:5]: Yes! let me change it

File: README.md [hunk 3]
```diff
@@ -3,12 +3,19 @@ A repo for solidity snippets and helpful resources
 - [What is a Smart Contract?](https://ethereum.org/en/developers/docs/smart-contracts/)
 
 ## 🔍 Cryptography
-- [What is a SHA-256](https://academy.bit2me.com/sha256-algoritmo-bitcoin/)
-- [HASH](https://academy.bit2me.com/que-es-hash/)
-- [NONCE?](https://academy.bit2me.com/que-es-nonce/)
+- [What is a SHA-256](https://academy.bit2me.com/en/que-es-hash/)
```
Comment (PratikGarai) [inline README.md:6]: sha256 link takes to page describing "hash"

File: README.md [hunk 4]
```diff
@@ -3,12 +3,19 @@ A repo for solidity snippets and helpful resources
...
+- [HASH](https://academy.bit2me.com/en/que-es-hash//)
```
Comment (PratikGarai) [inline README.md:7]: invalid link

**Review by PratikGarai — CHANGES_REQUESTED** [review]
Some links invalid

**Pull request merged**